pgctl/cli.py

```diff
@@ -517,6 +520,10 @@ def parser():
         '--json', action='store_true', default=False,
         help='output in JSON (only supported by some commands)',
     )
+    parser.add_argument(
+        '--force', action='store_true', default=False,
+        help='forcefully terminate zombie processes that prevent services from starting/stopping',
```

Comment:
there's some messageing already in pgctl calling them "run-away processes", I think.